### PR TITLE
Humanizer modules: curve, trader joe, across, kyberswap, 1inch

### PR DESCRIPTION
**Pull request by JIOjosBG**
- [x] Fixes entry point 0.7.0 humanization
- [x] Basic CURVE humanizer
- [x] Trader Joe, only  swapping
- [x] Accross bridge
- [x] Kyberswap, only swapping
- [x] [Not merged multiswaps one token pair](https://github.com/AmbireTech/ambire-app/issues/1688#issuecomment-2163176858)
- [x] swap on 1inch

most taken from:
 - https://github.com/AmbireTech/ambire-app/issues/1688

### CODE REVIEW

**Review by Ivshti — APPROVED** [review]

File: src/libs/humanizer/modules/Curve/cureve.test.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
+import humanizerInfo from '../../../../consts/humanizer/humanizerInfo.json'
+import { AccountOp } from '../../../accountOp/accountOp'
+import { HumanizerMeta } from '../../interfaces'
```
Comment (PetromirDev) [inline src/libs/humanizer/modules/Curve/cureve.test.ts:3]: typo in the name of the test

File: src/libs/humanizer/modules/1Inch/index.ts [hunk 2]
```diff
@@ -0,0 +1,82 @@
+import { Interface, ZeroAddress } from 'ethers'
+
+import { AccountOp } from '../../../accountOp/accountOp'
```
Comment (PetromirDev) [inline src/libs/humanizer/modules/1Inch/index.ts:3]: Token in not detected in humanization (1inch)
image

File: src/libs/humanizer/index.ts [hunk 3]
```diff
@@ -44,6 +49,11 @@ export const humanizerCallModules: HumanizerCallModule[] = [
   genericErc721Humanizer,
   gasTankModule,
   uniswapHumanizer,
+  curveModule,
+  traderJoeModule,
+  KyberSwap,
+  AcrossModule,
+  OneInchModule,
```
Comment (PetromirDev) [inline src/libs/humanizer/index.ts:56]: Not related but the displayed label for approvals is wrong. 
image

File: src/libs/humanizer/modules/1Inch/index.ts [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+      const { token: tokenArg, amount } = iface.parseTransaction(call)!.args
+      const token = `0x${BigInt(tokenArg).toString(16).padStart(40, '0')}`
+
+      return [getAction('Swap'), getToken(oneInchAddressParser(token), amount)]
+    },
+    [iface.getFunction(
+      'unoswap3(uint256 token,uint256 amount,uint256 minReturn,uint256 dex,uint256 dex2,uint256 dex3)'
+    )?.selector!]: (call: IrCall) => {
+      const { token: tokenArg, amount } = iface.parseTransaction(call)!.args
+      const token = `0x${BigInt(tokenArg).toString(16).padStart(40, '0')}`
```
Comment (PetromirDev) [inline src/libs/humanizer/modules/1Inch/index.ts:69]: this could perhaps be turned into a helper

**Review by PetromirDev — CHANGES_REQUESTED** [review]

File: src/libs/humanizer/modules/1Inch/index.ts (hunk 2 above)
Comment (JIOjosBG) [reply src/libs/humanizer/modules/1Inch/index.ts:3]: Yes, happens only on some 1inch swaps. We will go forward with it

**Pull request merged**